### ISSUE TITLE
Details on hardware

### ISSUE THREAD

**Issue by rgabrielson11** [owenb321/hampton-bay-fan-mqtt#2]
Hello,

Ive picked up the D1 Mini and the CC1101 you reccomended, wired and programmed but am not having any luck receiving or sending data.  It connects to MQTT fine and appears to be just sitting.  Tried a different CC1101 and even a different 8266 board.  Can you give more info on the exact fan remote you are controlling?  I am moving from Sonoffs to these due to reliability issues.

Thanks

**Comment by MACscr** [comment]
Sure you have the right frequency set?

**Comment by rgabrielson11** [comment]
yes, ive tried both using the frequency as it comes (303.631) and the frequency listed on the remote (303.9).  two different brands of remotes/receiver units, two different 8266 boards and two different receivers.  connects to MQTT and then nothing else.  Assuming at this point there is some issue with the program. Would love to hear if anyone other than OP has got this going as I was super excited to get rid of my sonoff ifans.

**Comment by MACscr** [comment]
I get the transceiver he recommends in the mail tomorrow and will give it a try. I already have a SDR setup that receives at 303mhz just fine and used with RTL_433, so I can easily verify frequencies, etc. 

- Mark

**Comment by rgabrielson11** [comment]
Cool thanks.  I am guessing its possible both CC1101s I received were bad,  so will wait for your results.

**Comment by owenb321** [comment]
The remote I captured the signals from is this [Anderic Replacement FAN-9T](https://www.amazon.com/Anderic-Replacement-REVERSE-Thermostatic-Control/dp/B06W579SHC) one. This model does not include light dimming, so if yours does, that might make a difference.

The code as-is should spit out anything the rc-switch picks up on that frequency over serial. Have you tried monitoring serial, or just MQTT?

**Comment by rgabrielson11** [comment]
Tried both serial and Mqtt. Mqtt gives birth message so working to there.  Serial looks like it’s not ever getting past my switch.avaliable  

Thinking now it may be my receiver. Before I move ahead I’ll see how others fare. Thanks 


---sent from my iPhone

**Comment by MACscr** [comment]
Ugh, amazon failed to notify me that the transceiver got backordered and now wont be delivered until the 29th. I dont see another one exactly like that on prime near the same price, so i guess i have to wait. Love how they never said anything about a delay and even up to contacting them today, it still said i should expect it today.

**Comment by MACscr** [comment]
Just got mine in the mail and got it all setup. no luck here either. Should be more to configuring this than whats listed i would think. Monitoring serial, but no output. Still not sure this hardware can do 303mhz when its not listed in its specs.

**Comment by MACscr** [comment]
Yep, i just changed the frequency to 315mhz and tried a 315mhz remote i have and it picked it up right away. Ugh.

**Comment by rgabrielson11** [comment]
I picked up a few of these, which are 433.  going to give those a try this weekend.  https://smile.amazon.com/gp/product/B07V27F64Q/ref=ppx_yo_dt_b_asin_title_o06_s00?ie=UTF8&psc=1

**Comment by MACscr** [comment]
Oh, i have 433mhz working perfectly with a sonoff rf bridge already. Bought this specifically for 303mhz. I just tested it with 433 and that worked too. Had to be exact though. 433 didnt work, but 433.92 did.

**Comment by LSatan** [comment]
Here is the data sheet for the CC1101. You can see the supported frequencies on the first page.
https://www.ti.com/lit/gpn/cc1101

Many of the cc1101 modules have very imprecise crystals. therefore the frequency differs from module to module. if you own SDR you should calibrate your module.
https://github.com/LSatan/CC1101-Debug-Service-Tool/tree/master/Calibrate_frequency
Read instructions of readme.md from this folder.

Regards
